docs/pages/models/triggers.mdx

```diff
@@ -244,8 +244,15 @@ The following arguments are available for all triggers:
   in this argument being `true`.
 - `before`: Only available for triggers of type [Following](#Following).
 - `after`: Only available for triggers of type [Following](#Following).
-- `options.implicit`: Indicates whether the query was automatically/implicitly generated
-  by a trigger instead of arriving from the outside.
+- `options.parentTrigger`: Indicates whether the query was generated by a trigger instead
+  of arriving from the outside. If it was, the property will contain its model slug and
+  its trigger type.
+- `options.headless`: Indicates whether the query is coming from "the outside", meaning
+  either from the [client-side of Blade](/hooks#usemutation-client) or the auto-generated
+  [REST API](https://blade.im/api-routes#api-routes). In those cases, you should consider
```

Comment:
Why is one of these urls relative (`/...`) and the other absolute (`https://blade.im/....`)?

Reply:
oops, great catch! fixing!